ydb/core/mind/hive/monitoring.cpp

```diff
@@ -3974,6 +4005,57 @@ class TTxMonEvent_Storage : public TTransactionBase<THive> {
     }
 };
 
+class TTxMonEvent_Subactors : public TTransactionBase<THive> {
+public:
+    const TActorId Source;
+    THolder<NMon::TEvRemoteHttpInfo> Event;
+    TMaybe<TSubActorId> SubActorToStop;
+
+    TTxMonEvent_Subactors(const TActorId &source, NMon::TEvRemoteHttpInfo::TPtr& ev, TSelf *hive)
+        : TBase(hive)
+        , Source(source)
+        , Event(ev->Release())
+    {
+        SubActorToStop = TryFromString<TSubActorId>(Event->Cgi().Get("stop"));
+    }
+
+    TTxType GetTxType() const override { return NHive::TXTYPE_MON_SUBACTORS; }
+
+    bool Execute(TTransactionContext &txc, const TActorContext& ctx) override {
+        Y_UNUSED(txc);
+        if (SubActorToStop) {
+            Self->StopSubActor(*SubActorToStop);
+        }
+        TStringStream str;
+        RenderHTMLPage(str);
+        ctx.Send(Source, new NMon::TEvRemoteHttpInfoRes(str.Str()));
+        return true;
+    }
+
+    void Complete(const TActorContext& ctx) override {
+        Y_UNUSED(ctx);
+    }
+
+    void RenderHTMLPage(IOutputStream& out) {
+        out << "<script>$('.container').css('width', 'auto');</script>";
+        out << "<table class = 'table simple-table2'>";
```

Comment:
you won't have "simple-table2" by default. it is defined manually in other methods.